images/src/generate.ts

```diff
@@ -81,14 +81,13 @@ async function download(client: gptscript.GPTScript, imageUrl: string): Promise<
     responseType: 'arraybuffer'
   })
   let content = Buffer.from(response.data, 'binary')
-
   // Convert the image to webp format
-  content = await sharp(content).webp({ quality: 100 }).toBuffer()
+  content = Buffer.from(await sharp(content).webp({ quality: 100 }).toBuffer())
 
   // Generate a SHA-256 hash of the imageURL to use as the filename
   const filePath = `generated_image_${createHash('sha256').update(imageUrl).digest('hex').substring(0, 8)}.webp`;
 
-  await client.writeFileInWorkspace(`${threadId ? 'files/' : ''}${filePath}`, content);
+  await client.writeFileInWorkspace(`${threadId ? 'files/' : ''}${filePath}`, content.buffer);
```

Comment:
Should this still be `files/` or should it also be changed to `file/`?

Reply:
No, this shouldn't change